### ISSUE TITLE
DateRangePicker not highlighting date range when dates are set using setState

### ISSUE THREAD

**Issue by tylerhaugen-stanley** [chrisdavies/tiny-date-picker#81]
Hi Chris,

Great to see that you are actively working on this and responding promptly! 

As the title describes, when setting the start and end date using setState, the highlighted date range does not render until I mouseover one of the dates. Please see the attached pictures to see what I mean. 

Here is how i'm initializing the dateRangePicker & setting the state. 
```
this._dateRangeInstance = new DateRangePicker.DateRangePicker(dateRangePickerElement, {
    startOpts: {
        mode: 'dp-permanent',
        hilightedDate: new Date(), // Removes the 'current day' border in the top right
        min: new Date() // Disabled any dates that have already passed
    }
});

this._dateRangeInstance.setState({
    start: new Date('04/12/2018'),
    end: new Date('05/12/2018')
});
```

Here is an image before mouseover:
![image](https://user-images.githubusercontent.com/11413202/38893405-379ee90c-4258-11e8-877a-f594a68def33.png)

And here is an image after:
![image](https://user-images.githubusercontent.com/11413202/38893425-46864b22-4258-11e8-826b-af8538848f2d.png)

**Comment by chrisdavies** [comment]
Interesting. Sorry on the late response. I've been out of town and offline!

This week, I'll take a look at a number of issues around the date-range picker. Thanks for reporting.

**Comment by tylerhaugen-stanley** [comment]
No problem. Thanks for actively responding to this, I'm happy to provide more help if needed.

**Comment by chrisdavies** [comment]
Should be fixed in the latest release (3.2.5). Thanks for the help!

**Comment by tylerhaugen-stanley** [comment]
Looks great! Thanks Chris!
